File: independent-projects/tools/codestarts/examples/quarkus/my-example/java/src/main/java/org/acme/example/MyHelloExample.java
package org.acme.example;

import javax.ws.rs.GET;
import javax.ws.rs.POST;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("/my-hello-example")
public class MyHelloExample {

    @GET
    @Produces(MediaType.TEXT_HTML)
    public String hello() {
        return "My Example Hello Quarkus Codestart";
    }

}

File: independent-projects/tools/codestarts/examples/quarkus/my-example/kotlin/src/main/kotlin/org/acme/example/MyHelloExample.kt
package org.acme.example

import java.util.*
import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.Produces
import javax.ws.rs.core.MediaType

@Path("/my-hello-example")
class MyHelloExample {

    @GET
    @Produces(MediaType.TEXT_HTML)
    fun hello(): String {
        return "My Example Hello Quarkus Codestart"
    }
}
